src/hooks/useToggleFormField.tsx

```
@@ -50,14 +50,15 @@ function useToggleFormField(
   schema: Schema,
   isRequired: boolean,
 ): {
-  inputMode: FieldInputMode;
+  inputMode: FieldInputMode | undefined;
   onOmitField: () => void;
 } {
   const [parentFieldName, fieldName] = getFieldNamesFromPathString(name);
   const { values: formState, setValues: setFormState } =
     useFormikContext<ModComponentFormState>();
-  const parentValues: UnknownObject =
-    getIn(formState, parentFieldName) ?? formState;
+  const parentValues: UnknownObject = parentFieldName
+    ? getIn(formState, parentFieldName, formState)
+    : formState;
```

Do you think we might want to add a little bit of safety here around the `getIn()` lookup? If it fails and returns undefined or whatever, should we throw an explicit error, "parentFieldName not found in form state" or something like that? Pretty sure this only passes strict null checks because Formik is lazy and uses `any` everywhere, right? Technically the return type of `getIn()` should include `| undefined` or whatever.

---

We're not though. The third arg for `getIn` is the default.